### PR TITLE
Modifies the Illumina test dataset samples for the metagenomics workflow with a samplesheet

### PR DESCRIPTION
**Pull request by elizabethmcd**
- Modifies the directory structure so the main directory is the name of the `metagenomics` workflow and sub-directories are organized by data type. Currently only have test data for Illumina PE short reads from cheese communities, working on long read data for PacBio and ONT
- Provides a samplesheet input for the modified metagenomics workflow to now take samplesheets instead of paths

### CODE REVIEW

File: metagenomics/illumina/samplesheet.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿sample,fastq_1,fastq_2
+comm_1,https://github.com/Arcadia-Science/test-datasets/tree/main/metagenomics/illumina/comm_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/tree/main/metagenomics/illumina/comm_1_subsampled_2.fq.gz
+vir_1,https://github.com/Arcadia-Science/test-datasets/tree/main/metagenomics/illumina/vir_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/tree/main/metagenomics/illumina/vir_1_subsampled_2.fq.gz
```
Comment (taylorreiter) [inline metagenomics/illumina/samplesheet.csv]: nit: missing new line

Comment (taylorreiter) [reply metagenomics/illumina/samplesheet.csv]: I think these are the wrong links. you have to get the link you could use from the "download" button to make sure you're going to the raw file, not the file previewer

**Review by taylorreiter — CHANGES_REQUESTED** [review]

File: metagenomics/illumina/samplesheet.csv (hunk 1 above)
Comment (elizabethmcd) [reply metagenomics/illumina/samplesheet.csv]: woops just fixed with latest commit

**Review by taylorreiter — APPROVED** [review]
LGTM!

**Pull request merged**